app/services/streaming/streaming.ts

```diff
@@ -1373,9 +1373,9 @@ export class StreamingService
           [OS.Mac]: filename,
           [OS.Windows]: filename.replace(/\//, '\\'),
         });
-        this.recordingModeService.actions.addRecordingEntry(parsedFilename);
-        this.markersService.actions.exportCsv(parsedFilename);
         this.recordingModeService.addRecordingEntry(parsedFilename);
+        // this.recordingModeService.actions.addRecordingEntry(parsedFilename);
```

Comment:
is there a reason we aren't using the actions pattern here?

Reply:
Updated